src/it/resources/synch/sampleSynchRequest.json

```diff
@@ -25,7 +25,7 @@
       },
       "childResources": [
         {
-          "apiVersion": "extensions/v1beta1",
+          "apiVersion": "networking.k8s.io/v1beta1",
```

Comment:
Is this correct? Why not: `networking.k8s.io/v1`

Reply:
```
kubectl get ingress now prefers the networking.k8s.io/v1 over extensions/v1beta1 (deprecated since v1.14). To explicitly request the deprecated version, use kubectl get ingress.v1beta1.extensions. (#94309, @liggitt) [SIG API Machinery and CLI]

```

Reply:
I think `networking.k8s.io/v1` wasn't working when I tried it. It's a few weeks ago now though.

Reply:
Ah looks like it jsut went stable in 1.19 maybe?
https://kubernetes.io/docs/concepts/services-networking/ingress/